cli/src/main.rs

```diff
@@ -9,20 +10,25 @@ fn main() {
         .version(crate_version!())
         .author(crate_authors!())
         .setting(AppSettings::ArgRequiredElseHelp)
-        .subcommand(login_sub_command());
+        .subcommand(login_sub_command())
+        .subcommand(package_sub_command());
 
     let matches = app.get_matches();
     match matches.subcommand_name() {
         Some("login") => println!("loginnnn",), // login
+        Some("package") => package::create_package("./".to_string(), "/tmp/package.tar.gz".to_string()), // login
         _ => (),                                // Either no subcommand or one not tested for...
     };
 }
 
-
 fn login_sub_command<'a, 'b>() -> clap::App<'a, 'b> {
     return SubCommand::with_name("login").about("Login to buff").arg(
         Arg::with_name("username")
             .short("u")
             .help("Case sensative username"),
     );
 }
+
+fn package_sub_command<'a, 'b>() -> clap::App<'a, 'b> {
+    return SubCommand::with_name("package").about("package");
```

Comment:
Lets add help and other niceties to this..

Reply:
removed it, as package won't be a real command IMHO, so just used in the test atm